.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    timeout-minutes: 5
```

Comment:
aggressive. what's its average? I guess this is PER job not for the whole combined thing so maybe i'm overestimating.

Reply:
Even when it has to load the whole NVD db from scratch it doesn't top 5 mins I believe (we'll see if this merges before that), but maybe we tune up if we find we need to

Reply:
OK, it's now set to 10 mins